src/main/java/graphql/nadel/NadelExtraInput.java

```diff
@@ -0,0 +1,25 @@
+package graphql.nadel;
+
+import graphql.Internal;
+
+import java.util.concurrent.ForkJoinPool;
+
+@Internal
+public class NadelExtraInput {
```

Comment:
Naming

`NadelOperationParams`
`NadelExecutionParams`

its not really input in the "query" sense like the other.